start-finalSetup02Modpack

```diff
@@ -111,7 +112,7 @@ esac
 fi
 
 if [[ "${GENERIC_PACK}" ]]; then
-  if isURL "${GENERIC_PACK}"; then
+  if [ -f /data/.generic_pack.sum && isURL "${GENERIC_PACK}"]; then
```

Comment:
I should have included some comments in the code here. I'll explain my logic, but I can also see the need for the change you're proposing here:

My theory was that the generic pack at the given URL might have changed since last downloaded/installed. So, at the expense of downloading the generic pack archive on every startup, it guarantees that the latest is always in use by comparing the checksum of the downloaded archive with the `.generic_pack.sum`.

For large modpacks that download expense could be quite significant, so I'd propose that you add a check for a new environment variable, such as `SKIP_GENERIC_PACK_CHECKSUM` to have it skip the download when the checksum file is present.

Reply:
I was just thinking more about this change and was thinking it would be nice if there was a web service out there that given a URL would return a checksum for the file at that URL. I did a quick search and couldn't find anything, so I might have to implement one if only for the fun of it.

Reply:
oh you mean something like https://developers.virustotal.com/reference#file-report? Where one can post a url and get back a hash in either md5, sha256, sha1, plus the scan date for that hash.

Reply:
Very helpful. It looks like `/url/report` (https://developers.virustotal.com/reference#url-report) might be exactly what I was thinking since it could be given the remote pack URL to analyze.

Reply:
hey the featureyou want already exists. It is the etag mechanism and is used extensively on the web for intermediate caching.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag
Check the headers!